WMFModel/WMFArticlePreview.h

```diff
@@ -1,22 +0,0 @@
-#import <Mantle/Mantle.h>
-@import CoreLocation;
-
-@interface WMFArticlePreview : MTLModel
```

Comment:
Probably going to have to undelete all of these to migrate, right? Or do you plan to rebuild them all?